src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -473,6 +508,7 @@ public function render()
 
 		$cssClasses = implode(' ', $this->breadcrumbsCssClasses);
 		return '<'. $this->listElement .' class="' . $cssClasses .'">' 
+		       . (!is_null($this->icon) ? '<i class="'. $this->icon .'"></i>' : '')
```

Comment:
Hey, I've just noticed - this actually adds an `<i>` element as a direct child of an `<ul>` (or `<ol>` if the users sets that) - which isn't valid. So we have to figure out a different strategy to accomplish this - do you have any ideas what the best way to do that would be?